README.md

```diff
@@ -1,2 +1,35 @@
 # rate-limiting-log-proxy
-A syslog-compatible log proxy that limits based on syslog tag
+
+This proxy is designed to rate limit logs coming from docker in a per-container fashion.  It does this by acting like a syslog server, applying rate limiting, then forwarding logs to a local journald server.
+
+The proxy tries to replicate as close as possible the log format coming out of the docker journald logging driver.
+
+## Building
+
+To build, just:
+
+```bash
+$ go build
+```
+
+## Tests
+
+Run tests with
+
+```bash
+$ go test ./...
+```
+
+## Running
+
+TODO: currently ports and destination of docker client are hardcoded, will pull this out into flags later
+
+## Configuring docker to send to proxy
+
+To send to this proxy, you should configure your docker daemon with:
+
+`--log-driver=syslog`
+`--log-opts syslog-format=rfc3164`
+`--log-opts syslog-address=udp://localhost:10514`
```

Comment:
would be curious to know if unixgram:// works here. might be preferable to udp-over-loopback, but I'd need to do a bit of research to be able to really justify that statement.